spec/refresher_spec.rb

```diff
@@ -5,78 +5,78 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Array).to be true
+      expect(Array.new.instance_of? Array).to be true
     end
 
     it "Yes, it is a Object, but which class" do
-      expect({}.is_a? _fill_in_object_).to be true
-      expect(Hash.new.instance_of? _FILL_ME_IN_).to be true
+      expect({}.is_a? Hash).to be true
+      expect(Hash.new.instance_of? Hash).to be true
     end
   end
 
   context "guess the type" do
     it "Yes, it is a Object, but which" do
-      expect(:class.is_a? _fill_in_object_).to be true
+      expect(:class.is_a? Symbol).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect("module".is_a? _fill_in_object_).to be true
+      expect("module".is_a? String).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.is_a? _fill_in_object_).to be true
+      expect(1.is_a? Fixnum).to be true
```

Comment:
[Ruby 2.4 removes the `Fixnum` (and `Bignum`) classes.](https://www.ruby-lang.org/en/news/2016/12/25/ruby-2-4-0-released/) `Integer` works on 2.4 and all previous versions since `Fixnum` and `Bignum` were subclasses of `Integer`.